File: services/sinodal-service/src/main/java/com/sgptt/SinodalService/repository/ProfessorRepository.java
package com.sgptt.SinodalService.repository;

import com.sgptt.SinodalService.entity.Professor;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface ProfessorRepository extends JpaRepository<Professor, Long> {
}

File: services/protocols-service/src/main/kotlin/com/sgptt/protocolsservice/repository/ProfessorRepository.kt
package com.sgptt.protocolsservice.repository

import com.sgptt.protocolsservice.repository.entity.Professor
import org.springframework.data.jpa.repository.JpaRepository

interface ProfessorRepository : JpaRepository<Professor, Long> {
	fun existsProfessorByProfessorNumber(number: String): Boolean
	fun findByProfessorNumber(number: String): Professor
}